common/src/main/java/com/adaptivescale/rosetta/common/TranslationMatrix.java

```diff
@@ -20,6 +21,11 @@ public class TranslationMatrix {
     private static final String TRANSLATION_TABLE_NAME = "TRANSLATION";
     private static final String TRANSLATION_ATTRIBUTE_TABLE_NAME = "TRANSLATION_ATTRIBUTE";
 
+    private static final String DATABASE_TEMPLATE_TABLE_NAME = "DATABASE_TEMPLATE";
```

Comment:
@ailegion @jetroni we can put this implementation in a new class or rename the TranslationMatrix to a something more general.